aries_cloudagent/anoncreds/base.py

```diff
@@ -136,6 +136,12 @@ async def get_revocation_list(
     ) -> GetRevListResult:
         """Get a revocation list from the registry."""
 
+    @abstractmethod
+    async def get_revocation_registry_delta(
+        self, profile: Profile, rev_reg_def_id: str, timestamp: None
+    ) -> Tuple[dict, int]:
+        """Get a revocation registry delta."""
+
```

Comment:
Delta's are an Indy AnonCreds Method specific implementation detail. I don't think this belongs in the abstract base.